eduvpn/nm.py

```diff
@@ -19,18 +19,16 @@
 from eduvpn.utils import run_in_glib_thread
 from eduvpn.variants import ApplicationVariant
 
-gi.require_version("NM", "1.0")  # noqa: E402
+from gi.repository import GLib  # type: ignore
```

Comment:
probably something like `gi.require_version("GLib", "2.0")` or such should also be inserted here...

Reply:
Whatever works is fine by me :)